### DISCUSSION TITLE
Failed to load resource

### DISCUSSION THREAD

**Discussion by barterkamp** [vercel/community#1510] [Help] [answered]
### Summary

Hello everyone 👋🏻

I'm confused. My site works perfectly on my local host (all the paths are correct). The build is successfully deployed, but when I open the site my styles are messed up and I see the following errors in my console: 

![Screenshot 2023-02-13 at 10 11 49](https://user-images.githubusercontent.com/84913017/218425003-1e57b532-8dd5-47c7-bf0f-ff13f9eafd2f.png)

Does anyone know how to solve this?

**Comment by amyegan** [comment]
Hi, Bart! 👋 We can take a look and help you debug if you share a [minimal reproduction](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) (or your repo, it it is public). An example deployment link can also be helpful for debugging

**Reply by barterkamp** [reply]
Hi Amy,

Sorry about that. The repo can be found [here](https://github.com/barterkamp/dictionary-app).
I'm using Alpine JS and Tailwind for this app, so perhaps I have to do something manually with the build? 

I have the following scripts in de package.json but I don't know how to run these manually with an override (it doesn't work)

"build-css": "npx tailwindcss -i ./src/input.css -o ./public/styles.css --watch",
"build-js": "npx esbuild ./src/script.js --outfile=public/script.js --bundle --watch"

https://dictionary-app-wine.vercel.app/

**Comment by barterkamp** [comment, marked as answer]
I solved it. I accidentally also copy pasted the --watch flag into the build step.
